packages/asc-ui/src/theme/CompactThemeProvider.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { ThemeProvider, Theme } from '..'
+
+// These are the overrides for showing compact text for several typography types
+const overridesCompactTheme: Partial<Theme.ThemeInterface> = {
```

Comment:
Perhaps also write how we would implement this (like we can nest this within the default themeprovider)

Reply:
done